Overview.bs

```diff
@@ -1765,9 +1765,8 @@ The algorithm:
 
 1. Initialize <var>extended font subset</var> to be an empty font with no tables.
 
-2. Check that the [=Table keyed patch/format=] field in <var>patch</var> is equal to 'iftk', if it is
-    not equal then <var>patch</var> is not correctly formatted. Patch application has failed, return
-    an error.
+2. Check that the <var>patch</var> is valid according to the requirements in [[#table-keyed]] (requirements are marked with a
+    "must") and all [=TablePatch=]'s are contained within <var>patch</var>. If it is not return an error.
```

Comment:
"it" doesn't have a grammatical anaphoric reference here, changing to "Otherwise return an error." is one fix.